### PR TITLE
Enable RestructuredText in Vale

### PR DESCRIPTION
**Pull request by kquinn1204**
Verified Rst now has Vale support addressing issue https://github.com/Vale-at-Red-Hat/vale-at-red-hat/issues/75

### CODE REVIEW

**Comment by aireilly** [comment]
Looks good!

**Comment by themr0c** [comment]
Currently, these 2 disabled rules won't apply to markdown anymore, but will apply to rst. I believe it should still apply to markdown, but it's not relevant to disable the rules on rst:

RedHat.HeadingPunctuation = NO
RedHat.Headings = NO

NB: these disabled rules apply on markdown because I hijacked this format to validate ini files in a project. Maybe it makes no sense to propagate in this style and we can remove the lines.

File: .vale.ini [hunk 1]
```diff
@@ -52,6 +52,11 @@ TokenIgnores = (:[^\n]+: [^\n]+), (\[id=[^\n]+)
...
 BasedOnStyles = RedHat
 
+# Match Restructured text files
+[*.rst]
+# Styles to load, located in the `StylesPath` folder:
+BasedOnStyles = RedHat
+
 # Disabling rules (NO)
 RedHat.HeadingPunctuation = NO
 RedHat.Headings = NO
```
Comment (themr0c) [inline .vale.ini:62]:
```suggestion
# Disabling rules (NO)
; RedHat.HeadingPunctuation = NO
; RedHat.Headings = NO
```

**Comment by kquinn1204** [comment]
Hi @themr0c I have made the suggested change. Is it OK to merge ?

**Review by themr0c — APPROVED** [review]

**Pull request merged**